### ISSUE TITLE
[Support] Implement tracking for Image Downloads on Events and Resources

### ISSUE THREAD

**Issue by nedierecel** [department-of-veterans-affairs/va.gov-team#349]
## User Story
As a content producer for Outreach and Events, I want to understand how users are interacting with the outreach and events page. Staging page is on https://staging.va.gov/outreach-and-events/. Typical interactions include:

- PDF Downloads (Already tracked) 
- Image Downloads
- Outbound clicks to events hosted on non-VA platforms like Facebook (Needs tweaking to track Outbound links)
- Add to Calendar links

## Goal
Implement Google Analytics tracking via Google Tag Manager

## Acceptance Criteria
- [x] Implement image tracking
- [x] Update outbound link tracking rules to pick up clicks to Facebook and other third-party sites (the current rule is picking up outbound link clicks that don't contain va.gov, but social links like "Share on Facebook" reference va.gov URL)
- [x] Validated on staging, ensuring new rules also do not impact other areas of the site
- [X] Published in Google Tag Manager
- [x] Annotate changes in `WBC` and `All Production View`

## Definition of Done
- [x] **Attached to a Milestone** 
- [x] **Attached to an Epic** 
- [x ] **Labeled with Team**
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by nedierecel** [comment]
Next Steps:

- Agile6 team to share sample event page with other call-to-action types, like click to Eventbrite, call, or email
- Insights team to validate outbound tracking

**Comment by nedierecel** [comment]
@joanneesteban For your visibility, I'm waiting on sample test events to track other calls-to-action.

**Comment by nedierecel** [comment]
@joanneesteban Adding blocked while we wait for CMS team to deliver additional sample pages to test outbound event registrations

**Comment by nedierecel** [comment]
Removing blocked label after speaking with Jeff Brauer. The new staging URLs to review for Outbound clicks are:

- https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/
- https://staging.va.gov/outreach-and-events/events/va2k/
- https://staging.va.gov/outreach-and-events/events/women-veterans-tea/
- https://staging.va.gov/outreach-and-events/events/washington-county-va-outpatient-clinic-dedication/

**Comment by nedierecel** [comment]
`blocked` label added as Staging links to test other event types are down on staging. Jeff Brauer will provide an updated link with all possible outbound calls-to-action.

Jon and I will validate once it is live.

**Comment by nedierecel** [comment]
@jonwehausen Assigning you on this task as well. Can you help follow-up with Jeff Brauer to get an updated event page on staging with all the possible outbound calls-to-action?

**Comment by jonwehausen** [comment]
@jefflbrauer When you have a moment, would you be able to provide an update on getting these staging links live in order to validate our implementation? 

https://staging.va.gov/outreach-and-events/events/va2k/
https://staging.va.gov/outreach-and-events/events/women-veterans-tea/
https://staging.va.gov/outreach-and-events/events/washington-county-va-outpatient-clinic-dedication/

Or provide an update if there is other staging links that can be utilized. 

Thanks,
Jon Wehausen

**Comment by jefflbrauer** [comment]
Hi @jonwehausen you can see all the click options on this one event here:

- More details button (facebook URL)
- Call for more info tel link
- Email for more info mailto link 

https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/

Note- there's a defect with the phone number, but the link does click.

**Comment by jonwehausen** [comment]
# OBSERVATIONS & ADJUSTMENTS
- Email and Telephone outbound link clicks were not initially firing the Outbound link tag
- Created two new triggers connected to the Outbound Links tag: 

Trigger       | Logic
------------ | -------------
| MailTo Trigger | ![image](https://user-images.githubusercontent.com/48527022/60973396-349efd00-a2f6-11e9-8cf4-af5b6020111c.png)
| Telephone Trigger  | ![image](https://user-images.githubusercontent.com/48527022/60973420-41bbec00-a2f6-11e9-8ca4-b217445f122f.png)

@nedierecel Let me know if you would like retain this adjustment for email and telephone clicks. 

# VALIDATION & TESTING
## :heavy_check_mark: [Add to Calendar](https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/)
![image](https://user-images.githubusercontent.com/48527022/60969890-b9861880-a2ee-11e9-9dee-d6da0726f471.png)

## :heavy_check_mark: [Share on Facebook](https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/)
![image](https://user-images.githubusercontent.com/48527022/60969995-fa7e2d00-a2ee-11e9-9252-39b9cb344b6c.png)

## :heavy_check_mark: [Image Download 1](https://staging.va.gov/outreach-and-events/?q=benefit)
![image](https://user-images.githubusercontent.com/48527022/60970523-2bab2d00-a2f0-11e9-913e-164aef1c45e8.png)

## :heavy_check_mark:[Image Download 2](https://staging.va.gov/outreach-and-events/?q=benefit)
![image](https://user-images.githubusercontent.com/48527022/60970860-dfacb800-a2f0-11e9-8a86-4f821616d293.png)

## :heavy_check_mark:[Image Download 3](https://staging.va.gov/outreach-and-events/?q=benefit)
![image](https://user-images.githubusercontent.com/48527022/60970860-dfacb800-a2f0-11e9-8a86-4f821616d293.png)


## :heavy_check_mark:[Eventbrite Outbound Links](https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/)
![image](https://user-images.githubusercontent.com/48527022/60971001-34e8c980-a2f1-11e9-8897-9cbf4c3e1175.png)

## :heavy_check_mark:[WITH ADJUSTMENT - Telephone Link Click](https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/)
![image](https://user-images.githubusercontent.com/48527022/60973194-c5c1a400-a2f5-11e9-9f04-f18730eb9bd3.png)

## :heavy_check_mark:[WITH ADJUSTMENT - Email Link Click](https://staging.va.gov/outreach-and-events/events/honoring-alaskan-veterans/)
![image](https://user-images.githubusercontent.com/48527022/60973285-f43f7f00-a2f5-11e9-89da-264cdb0e1891.png)

**Comment by nedierecel** [comment]
@jonwehausen Thanks for confirming. This is ready for publishing in GTM.

**Comment by nedierecel** [comment]
@jonwehausen This has been published to Dev and Staging/Production (the latter 2 environments are currently combined). May you please annotate these changes in the GA`WBC Modernized` view before closing the issue?

**Comment by nedierecel** [comment]
@joanneesteban For reporting on Outreach and Events, these are the following naming conventions for Event Actions that we will expect to see for launch: 

- Add to Calendar: <cal invent goes here>
- PDF Downloaded: <PDF name goes here>
- Image Downloaded: <Image name goes here>
- Outbound Link Clicked: <URL goes here>

**Comment by nedierecel** [comment]
I've annotated the launch of Outreach and Events that is scheduled today. I will make edits if the schedule is shifted.
